Decred Wallet/Features/Wallets/WalletInfoTableViewCell.swift

```diff
@@ -69,7 +69,14 @@ class WalletInfoTableViewCell: UITableViewCell {
     }
     
     var numberOfAccountsToDisplay: Int {
-        return self.wallet != nil && self.wallet.displayAccounts ? self.wallet.accounts.count : 0
+        let isWalletAvailable = self.wallet != nil && self.wallet.displayAccounts
+        let importedTotalAmount = self.wallet.accounts.last?.dcrTotalBalance
+        let isImportedTotalAmountZero = !(importedTotalAmount! > 0.0)
+        if isWalletAvailable {
+            return isImportedTotalAmountZero ? self.wallet.accounts.count - 1 : self.wallet.accounts.count
+        } else {
+            return 0
+        }
```

Comment:
2 things
- the imported account is not necessarily identified as the last account in the wallet
- instead of making this comprehensive check here, it's easier and simpler to exclude all 0-balance imported accounts from the `WalletsViewController.loadWallets()` function instead. The following code snippet exemplifies how this can be accomplished:

https://github.com/raedahgroup/dcrios/blob/bb5490e511cb8a7b7ec74d9f9d779fc8131ee2a7/Decred%20Wallet/Features/Custom%20Dialogs/AccountSelectorDialog/AccountSelectorDialog.swift#L64-L65